rmw_zenoh_cpp/src/detail/rmw_data_types.cpp

```diff
@@ -35,137 +35,6 @@
 ///=============================================================================
 namespace rmw_zenoh_cpp
 {
-///=============================================================================
```

Comment:
I think we can remove from the headers

`<optional>`, `<sstream>` and `<string>`